### ISSUE TITLE
Using horizontal bars

### ISSUE THREAD

**Issue by 3DJakob** [buzinas/simple-scrollbar#62]
I can't seam to find a way to get this module to add horizontal bars instead of vertical ones. This feature would make it way more useful as the scrollbars on safari are hidden per default which means users easily miss horizontal scroll areas.

**Comment by Doublefacy** [comment]
I agree - I personally developing power bi custom visual that needs horizontal scrolling and not vertical.
I implemented this scrollbar and it only applies vertical scrollbar, where in fact I rather hide overflow than display vertical scrollbar.

I really hope someone can add support for horizontal scrollbar

**Comment by buzinas** [comment]
This library was created for vertical scrolls only. Happy to accept PRs though.
